### DISCUSSION TITLE
github my repo README.md markdown parse bug

### DISCUSSION THREAD

**Discussion by lcp0578** [community/community#55457] [Repositories] [answered]
### Select Topic Area

Bug

### Body

my repo https://github.com/lcp0578/cheat-sheets README.md markdown parse bug
![image](https://github.com/community/community/assets/10859621/db2740e7-592e-407b-a5ad-49f17d366278)
In my markdown editor Markdown Pad 2
![image](https://github.com/community/community/assets/10859621/0b8956d6-f7da-4320-af12-d82078c03502)

**Comment by lcp0578** [comment]
I think it is a bug.

**Comment by airtower-luna** [comment, marked as answer]
Looking at the raw markdown, different indentation is the problem: Where it works it's one tab per level (which is usually counted as 8 spaces), where it doesn't work it's 4 spaces. With high numbers this means your inner list isn't indented far enough, because it must be indented *at least* as much as the text of the numbered item.

For example, this is one of the items where the inner list doesn't work:

```markdown
116. [R语言](src/R/README.md)   
    - [rayshader](src/R/rayshader.md) R Package for 2D and 3D mapping and data visualization
```

The following should work. Mind that the only change is one extra space, so the `-` starting the inner list is indented just as far as the `[` at the start of the numbered text.

```markdown
116. [R语言](src/R/README.md)   
     - [rayshader](src/R/rayshader.md) R Package for 2D and 3D mapping and data visualization
```

[This example](https://github.github.com/gfm/#example-275) from the [GitHub Flavored Markdown Spec](https://github.github.com/gfm/) demonstrates the rule (together with the previous example).

**Reply by lcp0578** [reply]
thx, I recover it to tab, but it is don't work.
![image](https://github.com/community/community/assets/10859621/f80fd5a8-d3ca-4ef1-a3fd-0e16fd2d812f)
this is my commit log.
![image](https://github.com/community/community/assets/10859621/267322d9-3989-49b6-9909-4688c28d3ae2)

https://github.com/lcp0578/cheat-sheets/commit/8da04dd0d1a1f4c8d4537b82ad670604ab66ddca

**Reply by airtower-luna** [reply]
What happens if you actually use the correct number of spaces? The tab might be interpreted by the parser as less than 8 spaces (that's why I wrote "usually" above). I notice that above the problem 4 spaces would be enough, but where the problem occurs you'd need (at least) 5.

**Reply by lcp0578** [reply]
Yeah, it is correct when i change tab to 8 space,but i also think it is a bug.
The top of the document is all right, it use tab.
thanks again.

**Reply by airtower-luna** [reply]
At the top of the document the numbers are smaller, so smaller indentation still works. Based on what you see I assume the Markdown parser GitHub uses interprets a tab as 4 spaces, not 8.

Look at this example:

```
1. ABC
   - a
200. DEF
     - b
```

The inner lists (the one starting with `-`) must be indented to the same level as the text after the numbering (`ABC`/`DEF`). With `1.` three spaces are enough, with `200.` I need five.

**Reply by lcp0578** [reply]
thx, I got it.
